client_source/sqlplus/ut_run.sql

```diff
@@ -70,7 +85,12 @@ Parameters:
 
 whenever sqlerror exit failure
 whenever oserror exit failure
-conn &1
+
+define client_path="&1"
```

Comment:
I like the idea of keeping documentation in one place but it is of great value to have full syntax visible from within script.
It would be probably best to have documentation in both places, even if it means double maintenance.
Ideally we should have `ut_run.bat` and `ut_run.sh` report help on syntax if the syntax provided is not valid. Documentation in readme might be redundant then, as the scripts would be self-documenting.
Any thoughts on that?